nemo/export/trt_llm/nemo_ckpt_loader/nemo_file.py

```diff
@@ -59,6 +62,22 @@ def __init__(self, path: Union[Path, TarPath]) -> None:
         """No call to super().__init__ because it expects pure Path."""
         self.path = path
         self.storage_data = dict()
+'''
+
```

Comment:
@oyilmaz-nvidia probably a nitpick, maybe the old TarFileSystemReader can be removed just so code is clean.

Reply:
Actually good point, forgot to remove that